### ISSUE TITLE
Add Backdoor

### ISSUE THREAD

**Issue by nsa-official** [f-prime/zCoin#2]
I can't spawn coins to fool internet criminals and destroy the world. Please add a backdoor.

**Comment by f-prime** [comment]
Thanks for the heads up, I'll be adding that in the next patch

I can't spawn coins to fool internet criminals and destroy the world.
Please add a backdoor.

—
Reply to this email directly or view it on
GitHubhttps://github.com/Max00355/zCoin/issues/2
.

**Comment by sysr-q** [comment]
I agree, this isn't NSA approved until you can use it to fight ~~freedom~~ terror.
